pages/login.tsx

```diff
@@ -24,13 +28,28 @@ export const getServerSideProps: GetServerSideProps = async (ctx) => {
   const { host = "" } = ctx.req.headers;
   const protocol = /^localhost/g.test(host) ? "http" : "https";
   const baseUrl = `${protocol}://${ctx.req.headers.host}`;
+  const newClientId = await getSettingWithTime(NEW_CLIENT_ID_KEY);
   const clientId = await getSetting(CLIENT_ID_KEY);
   const executives = await countExecutives();
   if (!executives) {
     ctx.res.statusCode = 307;
     ctx.res.setHeader("Location", "/initialise");
     return EMPTY_PROPS;
   }
+  if (newClientId.value && newClientId.updatedAt) {
+    if (
+      DateTime.fromJSDate(newClientId.updatedAt).plus({ minutes: 5 }) >=
```

Comment:
Condition reversed?

Reply:
Don't think so. This means (updateAt + 5 mins) is after current time